src/Features/Core/Portable/GenerateMember/GenerateParameterizedMember/AbstractGenerateParameterizedMemberService.SignatureInfo.cs

```diff
@@ -228,9 +228,9 @@ private Accessibility DetermineAccessibility(bool isAbstract)
                 {
                     // Otherwise, figure out what accessibility modifier to use and optionally
                     // mark it as static.
-                    if (containingType.IsContainedWithin(State.TypeToGenerateIn) && !isAbstract)
+                    if (containingType.IsContainedWithin(State.TypeToGenerateIn))
                     {
-                        return Accessibility.Private;
+                        return isAbstract ? Accessibility.Protected : Accessibility.Private;
```

Comment:
this is less accessible than before as well.  Previously we would fall through to the public class below, but in an abstract class we'd prefer to generate a protected constructor as that is more idiomatic.